### DISCUSSION TITLE
Import via HTML Export

### DISCUSSION THREAD

**Discussion by dford16** [zim-desktop-wiki/zim-desktop-wiki#1898] [Q&A] [answered]
I have incorrectly backed up my notebook using the Export > Complete Notebook > HTML format > Each page exported to a separate file.

Is there any way to import the Notebook back into a fresh install of Zim Wiki?

**Comment by jaap-karssenberg** [comment]
Not an easy way, but all the data is there, so it would be very much
possible to write a script that converts the html back to wiki format.

Op ma 31 jan. 2022 18:09 schreef dford16 ***@***.***>:

**Comment by bruvellu** [comment, marked as answer]
You can convert individual HTML pages to Zim Wiki format using [Pandoc](https://pandoc.org/).

**Comment by jaap-karssenberg** [comment]
@Bruno - cool, wasn't aware of that one - that should get the job done
indeed

**Comment by dford16** [comment]
Thank you for the replies.

I moved from Qubes back to Arch, eventually had to reinstall Qubes and restore my old Qubes VMs form a backup, recover my notebook in the raw format and copy the folder over to usb. 

Ill keep Pandoc as a future method, I will use it for some other files I have.

Thanks for the help!
